src/lib/grammar.js

```diff
@@ -129,12 +129,13 @@ function buildRegex(map, keyEx, valEx) {
         '(?<' + keyEx + '>',
             keys,
         ')',
-        // value is optional
         '(?:\\(',
             '(?<' + valEx + '>',
                 GRAMMAR.VALUES,
             ')',
-        '\\))?',
+        '\\))',
+        // value is optional only for helper
+        keyEx === 'helper' ? '?' : ''
```

Comment:
@renatoi we can implement so that the diference is based on the syntax of the matcher e.g.

```is
[
    {
        "name": "Border",
        // No args or parenthesis required
        "matcher": "Bd",
        "styles": {
            "border-width": "1px",
            "border-style": "solid"
        }
    },
  
    {
        "name": "Line clamp",
        // Parentesis with args optional 
        "matcher": "LineClamp(?)",
        "styles": {
            "-webkit-line-clamp": "$0",
            "max-height": "$1"
        },
        ...
    },

   {
        "name": "Animation",
        // Parenthesis with args required
        "matcher": "Anim($)",
        "allowParamToValue": true,
        "styles": {
            "animation": "$0"
        }
   }
]
```

Let me know what you think about this suggestion.